src/test/java/com/fasterxml/jackson/databind/node/CustomArrayNode.java

```diff
@@ -0,0 +1,129 @@
+package com.fasterxml.jackson.databind.node;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.jsontype.TypeSerializer;
+
+public class CustomArrayNode extends BaseJsonNode
```

Comment:
could you remove the 'public'? just seems tidier to make it explicit that this is only for internal use

Reply:
I can but that would require me to move it from the `.node` subpackage to the parent package where the test resides. Should I do this?

Reply:
sure - no need for a new package for this test code